vips/vip-193/vip-193.ts

```diff
@@ -0,0 +1,202 @@
+import { ethers } from "hardhat";
+
+import { ProposalType } from "../../src/types";
+import { makeProposal } from "../../src/utils";
+
+const NEW_VBEP20_DELEGATE_IMPL = "0xc3279442a5aCaCF0A2EcB015d1cDDBb3E0f3F775";
+const ACCESS_CONTROL_MANAGER = "0x4788629ABc6cFCA10F9f969efdEAa1cF70c23555";
+const FAST_TRACK_TIMELOCK = "0x555ba73dB1b006F3f2C7dB7126d6e4343aDBce02";
+const CRITICAL_TIMELOCK = "0x213c446ec11e45b15a6E29C1C1b402B8897f606d";
+const PROTOCOL_SHARE_RESERVE = "0xCa01D5A9A248a830E9D93231e791B1afFed7c446";
+const COMPTROLLER = "0xfd36e2c2a6789db23113685031d7f16329158384";
+const NORMAL_TIMELOCK = "0x939bD8d64c0A9583A7Dcea9933f7b21697ab6396";
+
+interface AssetConfig {
+  name: string;
+  address: string;
+  reduceReservesBlockDelta: number;
+  holder: string;
+}
+
+export const CORE_MARKETS: AssetConfig[] = [
+  {
+    name: "vBUSD",
+    address: "0x95c78222B3D6e262426483D42CfA53685A67Ab9D",
+    reduceReservesBlockDelta: 105120000,
+    holder: "0xF977814e90dA44bFA03b6295A0616a897441aceC",
+  },
```

Comment:
I would move this market to the VIP-192

Then, these VIP's can be proposed following this order:
1) VIP-193 (only vTRXOLD and vTUSDOLD)
2) VIP-192 (rest of the markets in the Core pool)
3) VIP-191 (IL)

This way, the risk would be mitigated (starting only with vTRXOLD and vTUSDOLD).

We would need to rename the VIP's. Taking into account the current last proposed VIP to mainnet is 191, we would rename them this way:
1) VIP-193 -> VIP-192
2) VIP-192 -> VIP-193
3) VIP-191 -> VIP-194